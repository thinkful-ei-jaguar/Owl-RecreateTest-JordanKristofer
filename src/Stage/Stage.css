.Stage-participant{
  background-color: darkgrey;
  text-align: center;
  width: 50%;
  border: 1px solid blue;
}

.Stage-name {
  margin-top: 30px;
}

.Stage-avatar{
  margin-top: 30px;
  width: 30%;
  border-radius: 50%;
  border: 1px solid black;
}