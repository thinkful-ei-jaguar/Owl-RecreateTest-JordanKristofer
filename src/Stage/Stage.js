import React from 'react';
import './Stage.css'

class Stage extends React.Component{
    render(){
        return (
          <div className='Stage-participant' id={this.props.id}>
             <div className='Stage-name'>{this.props.name}</div>
             <img className='Stage-avatar' src={this.props.avatar} alt={this.props.name}/>
          </div>
        )
      }
    }

export default Stage;