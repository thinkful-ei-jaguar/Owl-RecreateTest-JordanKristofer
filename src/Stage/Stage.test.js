import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './Stage.js';


describe('<Stage />', () => {
  //Smoke Testing 
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Stage 
        // header='TEST HEADER'
      />, 
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  //Snapshot Testing 
  it('renders this UI as expected', () => {
    //render the component as JSON
    const tree = renderer
      .create(<Stage  
        />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  
});