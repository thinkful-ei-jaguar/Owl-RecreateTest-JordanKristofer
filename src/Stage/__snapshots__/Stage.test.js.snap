// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage /> renders this UI as expected 1`] = `
<div
  className="Participant-list"
>
  <div
    className="Participant"
  >
    <img
      alt="Koren Templeton"
      className="Person-image"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <div
      className="Person-info"
    >
      <p
        className="Person-name"
      >
        Koren Templeton
      </p>
      <div
        className="Person-activity"
      >
        <div
          className="Person-status"
        >
          active
        </div>
        <div
          className="Person-stage"
        >
          on stage
        </div>
      </div>
    </div>
  </div>
  <div
    className="Participant"
  >
    <img
      alt="Caty Flucker"
      className="Person-image"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <div
      className="Person-info"
    >
      <p
        className="Person-name"
      >
        Caty Flucker
      </p>
      <div
        className="Person-activity"
      >
        <div
          className="Person-status"
        >
          active
        </div>
        <div
          className="Person-stage"
        >
          in session
        </div>
      </div>
    </div>
  </div>
  <div
    className="Participant"
  >
    <img
      alt="Axe Kubicka"
      className="Person-image"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <div
      className="Person-info"
    >
      <p
        className="Person-name"
      >
        Axe Kubicka
      </p>
      <div
        className="Person-activity"
      >
        <div
          className="Person-status"
        >
          left session
        </div>
        <div
          className="Person-stage"
        >
          in session
        </div>
      </div>
    </div>
  </div>
  <div
    className="Participant"
  >
    <img
      alt="Frank Runciman"
      className="Person-image"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <div
      className="Person-info"
    >
      <p
        className="Person-name"
      >
        Frank Runciman
      </p>
      <div
        className="Person-activity"
      >
        <div
          className="Person-status"
        >
          active
        </div>
        <div
          className="Person-stage"
        >
          in session
        </div>
      </div>
    </div>
  </div>
  <div
    className="Participant"
  >
    <img
      alt="Ashla Attwool"
      className="Person-image"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <div
      className="Person-info"
    >
      <p
        className="Person-name"
      >
        Ashla Attwool
      </p>
      <div
        className="Person-activity"
      >
        <div
          className="Person-status"
        >
          active
        </div>
        <div
          className="Person-stage"
        >
          on stage
        </div>
      </div>
    </div>
  </div>
</div>
`;
